js/WebAPI/DocumentEditor.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: post
-title: webAPI reference for DocumentEditor
-description: webAPI reference for 
+title: WebAPI reference in JavaScript DocumentEditor Control | Syncfusion
```

Comment:
Update like this - WebAPI Reference in JavaScript Document Editor Control | Syncfusion

Reply:
resolved